.github/workflows/e2e-tests.yml

```diff
@@ -28,9 +28,6 @@ jobs:
           node-version: lts/*
           cache: 'npm'
 
-      - name: Update npm to latest
-        run: npm i -g npm
```

Comment:
Just in case this might affect tests — npm@6 and npm@7 handle `peerDependencies` differently; with npm@6 you have to install them manually; with `npm@7+`, they are installed automatically.

Reply:
Thanks for pointing that out. We're using the `lts` node version, which in this case is 16 and includes npm 7+.